lambdas/handlers/generate_lloyd_george_stitch_handler.py

```diff
@@ -0,0 +1,72 @@
+from boto3.dynamodb.types import TypeDeserializer
+from enums.lambda_error import LambdaError
+from enums.logging_app_interaction import LoggingAppInteraction
+from models.stitch_trace import StitchTrace
+from pydantic import ValidationError
+from services.lloyd_george_generate_stitch_service import LloydGeorgeStitchService
+from utils.audit_logging_setup import LoggingService
+from utils.decorators.handle_lambda_exceptions import handle_lambda_exceptions
+from utils.decorators.override_error_check import override_error_check
+from utils.decorators.set_audit_arg import set_request_context_for_logging
+from utils.lambda_exceptions import GenerateManifestZipException
+from utils.lambda_response import ApiGatewayResponse
+from utils.request_context import request_context
+
+logger = LoggingService(__name__)
+
+
+@set_request_context_for_logging
+@override_error_check
```

Comment:
Will this need @ensure_environment_variables similar to the generate_document_manifest_handler?